server/command.go

```diff
@@ -589,6 +596,17 @@ func (p *Plugin) executePromoteUserCommand(args *model.CommandArgs, parameters [
 	return p.cmdSuccess(args, "Account "+username+" has been promoted and updated the username to "+newUsername)
 }
 
+func (p *Plugin) executeStatusCommand(args *model.CommandArgs) (*model.CommandResponse, *model.AppError) {
+	if storedToken, err := p.store.GetTokenForMattermostUser(args.UserId); err != nil {
+		// TODO: We will need to distinguish real errors from "row not found" later.
+		return p.cmdSuccess(args, "Your account is not connected to Teams.")
```

Comment:
```suggestion
		return p.cmdSuccess(args, "Your account isn't connected to MS Teams.")
```

Reply:
Thank you! It's going to take me a while to accept that using contractions is OK :p